pkg/oc/admin/prune/imageprune/prune.go

```diff
@@ -914,13 +948,39 @@ func pruneStreams(
 			return err
 		})
 
+		if err == noChangeErr {
+			continue
+		}
 		if err != nil {
-			return fmt.Errorf("unable to prune stream %s: %v", streamName, err)
+			failures = append(failures, Failure{Node: streamNode, Err: err})
+		} else {
+			deletions = append(deletions, Deletion{Node: streamNode})
 		}
 	}
 
 	glog.V(4).Infof("Done removing pruned image references from streams")
-	return nil
+	return
+}
+
+func strenghtenReferencesFromFailedImageStreams(g genericgraph.Graph, failures []Failure) {
```

Comment:
s/strenghten/strengthen/

Reply:
You're not going to believe it, but my [offline dictionary](https://cs.glosbe.com/source/2721) contains both :smile:. I cannot find it online though so I'm going to trust you.

Reply:
Fixed